.github/ISSUE_TEMPLATE.md

```diff
@@ -10,11 +10,10 @@
 Include if relevant.
 
 Please execute `:let g:neomake_verbose = 3` to enable verbose output, which
-might help showing the issue already.
-Look at `:messages` to see them.
+might help showing the issue already.  Look at `:messages` for its output.
```

Comment:
Maybe replace the paragraph with:

```
Execute `:3verb Neomake` and look at `:messages` for its output.
```

And add at the top:

```
Please ensure you're using the latest version of Neomake before posting.
```

Don't give the other option since we can consider that to be more of a development variable.